spec.md

```diff
@@ -523,13 +518,295 @@ schema being used.
         "bullets": [
           "40 concurrent connections"
         ]
+      },
+      "schemas": {
+        "service_instance": {
+          "create": {
+            "parameters": {
+              "$ref": "osb.v2:///fakeservice/v1#BillingAccountParameters"
+            }
+          },
+          "update": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "propeties": {
+                "some-other-account": {
+                  "description": "Other fake account needed for update.",
+                  "type": "string"
+                },
+                "billing": {
+                  "$ref": "osb.v2:///fakeservice/v1#BillingAccountParameters"
+                }
+              }
+            }
+          }
+        },
+        "service_binding": {
+          "create": {
+            "parameters": {
+              "$ref": "osb.v2:///fakeservice/v1/bindings#AccountParameters"
+            }
+          }
+        }
       }
     }]
   }]
 }
 ```
 
 
+## JSON Schemas
+
+The JSON Schemas endpoint is used to define reusable definitions of JSON Schema
+Objects referenced throughout the [Catalog](#catalog-management). It will be the
+responsibility of the Platform to examine the response from the catalog to
+determine if the Broker is using `$ref` JSON Schemas. The Broker can signal to
+the Platform that JSON Schema components are supported by using the `osb_v2:`
+scheme. Components are a way to allow Platforms to only fetch sub-schemas.
+
+The root JSON Schema document can be found by getting the JSON Schemas
+endpoint without a `:component_path`.
+
+If the `osb_v2:` scheme is used by the broker, then the Platform MAY fetch the
+component JSON Schema directly using the following convention:
+
+Given: `osb_v2:///{:component_path}`, fetch `/v2/catalog/schemas/:component_path`
```

Comment:
just wondering... is there any way to achieve the same net results w/o introducing a new URL scheme? E.g. "foo" means its in the full schema doc but "/foo" means its available at /v2/catalog/schemas/foo".  Might not be a big enough distinction, but I'd like to brainstorm a bit to see if there are other options rather than introducing something that feels like a bit of a hack.